client/include/Maszyna.Diagnostics/Trace.hxx

```diff
@@ -0,0 +1,182 @@
+#ifndef __Maszyna_Diagnostics_Trace_hxx__
+#define __Maszyna_Diagnostics_Trace_hxx__
+
+#include <cstdint>
+
+namespace Maszyna
+{
+namespace Diagnostics
+{
+    namespace TraceLevel
+    {
+        enum Enum
```

Comment:
Basing on uint32 is nice, however promoting it to enum class would introduce additional scope. By this idiom, whole TraceLevel namespace acts like "enum" just because it allows to add ToString function to the same scope.
Possibilities:
 - promote to enum class, move ToString function to parent namespace / whatever
 - leave this enum as is